heat/core/tests/test_suites/basic_test.py

```diff
@@ -8,18 +9,50 @@
 
 
 # TODO adapt for GPU once this is working properly
-class BasicTest(TestCase):
+class TestCase(unittest.TestCase):
 
     __comm = MPICommunication()
     __device = None
 
     @property
     def comm(self):
-        return BasicTest.__comm
+        return TestCase.__comm
 
     @property
     def device(self):
-        return BasicTest.__device
+        return TestCase.__device
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Docsstring need to be updated here by the Raises section

Reply:
1. Raises added
2. I found that there is already an \_\_init\_\_.py in the test_suites folder.
3. Pytorch doesn't have an option to set the standard device. It is always CPU.

Reply:
2. Can you import the test cases there as well?
3. Understood.